ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/text/v1/text/clientlibs/site/js/textview.js

```diff
@@ -66,7 +66,13 @@
             updateValue(value) {
                 // html sets undefined value as undefined string in input value, hence this check is added
                 let actualValue = typeof value === "undefined" ? "" :  value;
-                const sanitizedValue = window.DOMPurify ? window.DOMPurify.sanitize(actualValue) : actualValue;
+                let sanitizedValue = window.DOMPurify ? window.DOMPurify.sanitize(actualValue, {
+                    ADD_ATTR: ['target', 'rel'] // Explicitly add target and rel to allowed attributes
+                }) : actualValue;
+
+                // Add rel="noopener noreferrer" to all links with target="_blank" to prevent xss
+                sanitizedValue = sanitizedValue.replace(/target="_blank"/g, 'target="_blank" rel="noopener noreferrer"');
```

Comment:
You don't need to do this, this is already added by AEM, we should not be adding such strings in the product code

Reply:
okay